src/component/Calendar/NextLink.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { calcDateRange } from './CalendarLogic';
+
+import styled from 'styled-components';
+
+export const NextLink = ({
+  linkInfo,
+  dateRange,
+  setCalendarOff,
+  redirectComponent,
+}) => {
+  const { stayLocation, linkUrl, linkButtonText } = linkInfo;
+  const stayInfo = calcDateRange(dateRange);
+
+  return (
+    <StyledLink
+      to={{
+        pathname: stayLocation ? linkUrl : redirectComponent,
+        state: { stayInfo },
+      }}
+    >
+      <NextStepButton
+        disabled={dateRange[1] !== null || stayLocation ? false : true}
```

Comment:
```suggestion
        disabled={dateRange[1] !== null || !stayLocation}
```
boolean값으로 표현하면 되는것들은 삼항연산자는 사용할 필요 없습니다

Reply:
```
(dateRange[1] !== null || stayLocation) ? false : true
```
이런 용도로 쓰인 코드였군요 그러면 괄호처리를 해주는게 좋을 것 같습니다!
|| 연산자 뒤에 있는 `stayLocation ? false : true` 이부분이 평가된 결과를 사용하는건지 stayLocation 값만 사용하는 지 읽는사람에게 혼동을 불러일으킬 여지가 있어보입니다.

```
!(dateRange[1] !== null || stayLocation)
```
저런 용도라면 위와 같이 삼항없이 표현 가능합니다!

Reply:
프리티어 친구가 괄호를 용인하지 않네요 ㅠㅠ..  대신 주석으로 해당 로직을 분할해서 설명하는 것은 어떨까요?

괜찮다고 하시면 다음 브랜치에서 적용해서 PR 올리겠습니다!